audience_manager/api.rst

```diff
@@ -1,175 +1,207 @@
+This code should be added just before you want to use the API. Additionally snippet has to be configured this way:
 
-* 'https://ppms.example.com/audiences/static/widget/audience-manager.api.min.js'
-* 'ppms.example.com'
-
-Snippet
-```````
-
-Code:
+- String ``XXX-XXX-XXX-XXX-XXX`` should be replaced with :term:`app ID` (e.g. ``efcd98a5-335b-48b0-ab17-bf43f1c542be``).
+- String ``ppms.example.com`` should be replaced with your PPMS domain name (please note that it's used in 2 places in
+  the snippet).
 
 .. code-block:: html
 
     <script>
         (function(a,d,g,h,b,c,e){a[b]=a[b]||{};a[b][c]=a[b][c]||{};a[b][c][e]=a[b][c][e]||function(){(a[b][c][e].q=a[b][c][e].q||[]).push(arguments)};var f=d.createElement(g);d=d.getElementsByTagName(g)[0];f.async=1;f.src=h;d.parentNode.insertBefore(f,d)})
         (window,document,"script","https://ppms.example.com/audiences/static/widget/audience-manager.api.min.js","ppms","am","api");
 
-        ppms.am.api('create', 'XXX-XXX-XXX-XXX-XXX', 'ppms.example.com');
+        ppms.am.api("create", "XXX-XXX-XXX-XXX-XXX", "ppms.example.com");
     </script>
 
-API function
-------------
+This code initializes API interface in following ways:
+
+    #. Creates a ``<script>`` tag that asynchronously loads Audience Manager API library.
+    #. Initializes global ``ppms.am.api`` command queue that schedules commands to be run when API library is loaded.
+    #. Schedules create API command on ``ppms.am.api``.
 
-Loading snippet creates following API function:
+You can use the API command queue (``ppms.am.api``) immediately after step 3.
+
+Command queue
+-------------
+Loading snippet creates following global function:
 
 .. function:: ppms.am.api(command, ...args)
 
-    JavaScript API interface.
+    Audience Manager API command queue.
 
     :param string command: Command name.
     :param args: Command arguments. Number of arguments and their function depend on command.
-    :returns: Nothing. All commands are prepared to be run asynchronously.
-
+    :returns: Commands are expected to be run asynchronously and return no value.
+    :rtype: undefined
 
 Commands
 --------
-Following commands work in context of current browser user profile. Additionally these commands require communication
-with server so they are asynchronous. Callback functions are used to provide response or error information.
+All commands work in context of current :term:`user`. Additionally they require communication with PPMS server and are
+asynchronous. Callback functions are used to provide response value or information about error.
 
-List audiences
-``````````````
-Gets list of user audiences.
+Get list of audiences user belongs to
+`````````````````````````````````````
+Gets list of :term:`audience` IDs :term:`user` belongs to.
 
 Code::
 
-    ppms.am.api('getAudiences', onFulfilled, onRejected);
+    ppms.am.api("getAudiences", onFulfilled, onRejected);
```

Comment:
How did we get to a point where `onSuccess` and `onError` are too mainstream to be used in docs? 😄

Reply:
I blame [Promises](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then).